src/modules/Accounts/components/AccountCard/AccountCard.js

```diff
@@ -1,88 +1,45 @@
 /* eslint-disable max-statements */
 import React from 'react';
 import { TouchableOpacity, View } from 'react-native';
-import { useNavigation } from '@react-navigation/native';
 import LinearGradient from 'react-native-linear-gradient';
-import i18next from 'i18next';
 
 import { useTheme } from 'contexts/ThemeContext';
-import { useAccountCanSendTokens } from 'modules/SendToken/hooks/useAccountCanSendTokens';
 import { P } from 'components/shared/toolBox/typography';
-import { useAuth } from 'modules/Auth/hooks/useAuth';
 import Avatar from 'components/shared/avatar';
-import { PrimaryButton } from 'components/shared/toolBox/button';
 import { stringShortener } from 'utilities/helpers';
 import { colors } from 'constants/styleGuide';
 import SwitchSvg from 'assets/svgs/SwitchSvg';
-import { useModal } from 'hooks/useModal';
-import MultiSignatureSvg from 'assets/svgs/MultiSignatureSvg';
 import CopyToClipboard from 'components/shared/CopyToClipboard/CopyToClipboard';
+import DiscreteModeComponent from 'components/shared/DiscreteModeComponent';
 import useAccountManagerModal from '../../hooks/useAccountManagerModal';
 
 import { useCurrentAccount } from '../../hooks/useCurrentAccount';
 
 import getAccountDetailsStyles from './AccountCard.styles';
-import TransactionError from '../../../Transactions/components/SignTransaction/SignTransactionError';
+import useLegacyAccount from '../../api/useLegacyAccount';
+import { fromBeddowsToLsk } from '../../../../utilities/conversions.utils';
+import Skeleton from '../../../../components/shared/Skeleton/Skeleton';
 
 export default function AccountCard({ account }) {
   const accountManager = useAccountManagerModal();
-  const modal = useModal();
-  const navigation = useNavigation();
 
   const [currentAccount] = useCurrentAccount();
 
-  const { data: accountSummary } = useAuth(account.address);
-  const accountIsMultisignature = accountSummary?.numberOfSignatures > 0;
+  const {
+    data: legacyAccount,
+    isLoading: isLoadingLegacyAccount,
+    isError: isErrorLegacyAccount,
+  } = useLegacyAccount();
 
   const { styles } = useTheme({ styles: getAccountDetailsStyles() });
 
   const isCurrentAccount = currentAccount.metadata.address === account.address;
 
-  const {
-    data: accountCanSendTokensData,
-    tokenName,
-    isLoading: isLoadingAccountCanSendTokens,
-    isError: isErrorAccountCanSendTokens,
-  } = useAccountCanSendTokens(currentAccount.metadata.address);
-
-  const disableSendTokenButton = isLoadingAccountCanSendTokens || isErrorAccountCanSendTokens;
+  const availableBalance =
+    legacyAccount?.token?.lockedBalances.reduce((acc, curr) => acc + Number(curr.amount), 0) ?? 0;
```

Comment:
this is lockedBalance (not available balance).
also, to be safe it should use the bigint
```suggestion
  const lockedBalance =
    legacyAccount?.token?.lockedBalances.reduce((acc, curr) => acc + BigInt(curr.amount), BigInt(0)) ?? BigInt(0);
```